src/main/java/de/teamlapen/vampirism/blocks/CoffinBlock.java

```diff
@@ -49,8 +49,8 @@
     public static final String name = "coffin";
     public static final EnumProperty<CoffinPart> PART = EnumProperty.create("part", CoffinPart.class);
     public static final BooleanProperty OCCUPIED = BooleanProperty.create("occupied");
-    public static final DirectionProperty FACING = HorizontalBlock.HORIZONTAL_FACING;
     private static final VoxelShape shape = makeShape();
+    private static final Map<PlayerEntity.SleepResult, ITextComponent> sleepResults = ImmutableMap.of(PlayerEntity.SleepResult.NOT_POSSIBLE_NOW, new TranslationTextComponent("text.vampirism.coffin.no_sleep"), PlayerEntity.SleepResult.TOO_FAR_AWAY, new TranslationTextComponent("text.vampirism.coffin.too_far_away"), PlayerEntity.SleepResult.OBSTRUCTED, new TranslationTextComponent("text.vampirism.coffin.obstructed"));
```

Comment:
i need to add one sleepresult

Reply:
not sure what you mean. Is this PR good to go or do you want to change something?

Reply:
now is good to go